packages/cubejs-api-gateway/test/index.test.ts

```diff
@@ -579,7 +579,10 @@ describe('API Gateway', () => {
       .expect(200);
     expect(res.body).toHaveProperty('cubes');
     expect(res.body.cubes[0]?.name).toBe('Foo');
+    expect(res.body.cubes[0]?.description).toBe('cube from compilerApi mock');
```

Comment:
If you're here, can you fix the warning above ^^^ too please?

Reply:
Done